packages/http-client/src/http-client-types.ts

```diff
@@ -121,9 +122,9 @@ export type HttpRequest = DestinationHttpRequestConfig & HttpRequestConfig;
 /**
  * @internal
  */
-export type ExecuteHttpRequestFn<ReturnT> = (
+export type ExecuteHttpRequestFn = (
   request: HttpRequest
-) => Promise<ReturnT>;
+) => Promise<HttpResponse>;
```

Comment:
[q] Is it intended? It seems like the information of `<ReturnT>` is lost after the change.